README.md

```diff
@@ -167,6 +170,28 @@ from the `txn` table when `commit()` returned an error is undefined.
 
 [Back to TOC](#table-of-contents)
 
+## Directives
+
+### lmdb_encryption_key_data
+
+**syntax:** *lmdb_encryption_key_data "YourPassword";*
+
+**context:** *main*
+
+Encrypt the lmdb database, note that set lmdb_encryption_key_data to enable encryption.
```

Comment:
This sentence seems a little unclear to me. Is there something to this second half that is missing?

> note that set lmdb_encryption_key_data to enable encryption.

Reply:
I agree with your opinion.